src/mbgl/map/transform_state.hpp

```diff
@@ -74,13 +77,17 @@ class TransformState {
     double zoomScale(double zoom) const;
     double scaleZoom(double scale) const;
 
+    constexpr explicit operator bool() const {
```

Comment:
Let's not use `operator bool()` for validity checks. If we need a validity check, name it `bool valid() const`. But first, try to eliminate the possibility of an invalid state.